examples/platform/telink/common/src/AppTaskCommon.cpp

```diff
@@ -227,6 +231,10 @@ CHIP_ERROR AppTaskCommon::InitCommonParts(void)
     emberAfEndpointEnableDisable(kNetworkCommissioningEndpointSecondary, false);
 #endif
 
+#ifdef CONFIG_CHIP_ICD_SUBSCRIPTION_HANDLING
```

Comment:
This is fine, but note that non-ICDs may also want to put a floor on their MaxInterval so they can hit their battery use targets.